content/graphql/subscriptions.md

```diff
@@ -33,7 +48,22 @@ type Subscription {
 }
 ```
 
-To filter out specific events based on context and arguments, set the `filter` property.
+To publish the event, use the `PubSub#publish` method. This is often used within a mutation to trigger a client-side update when a part of the object graph has changed. For example:
+
+```typescript
+@@filename(posts/posts.resolver)
+@Mutation(returns => Post)
+async addComment(
+  @Args({ name: 'postId', type: () => Int }) postId: number,
+  @Args({ name: 'comment', type: () => Comment }) comment: CommentInput,
```

Comment:
Can we use `@Args('comment', { type: () => Comment })` syntax here?

Reply:
✔️